index.php

```diff
@@ -1,56 +1,56 @@
 <?php
-/**
- * The main template file.
- *
- * This is the most generic template file in a WordPress theme
- * and one of the two required files for a theme (the other being style.css).
- * It is used to display a page when nothing more specific matches a query.
- * E.g., it puts together the home page when no home.php file exists.
- *
- * @link https://codex.wordpress.org/Template_Hierarchy
- *
- * @package xml-test
- */
 
-get_header(); ?>
+header('Content-Type: text/xml; charset=' . get_option('blog_charset'), true); //This sets the content type to a xml document and renders the proper format in a browser.
 
-	<div id="primary" class="content-area">
-		<main id="main" class="site-main" role="main">
+$link_cat = '';
+if ( !empty($_GET['link_cat']) ) {
```

Comment:
We need to sanitize user provided values. [See PHP Manual on Filter Input](http://php.net/manual/en/function.filter-input.php)